Directory.Build.props

```diff
@@ -0,0 +1,6 @@
+<Project>
+  <PropertyGroup>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <WarningsNotAsErrors>$(WarningsNotAsErrors);CS8981</WarningsNotAsErrors>
```

Comment:
@webwarrior-ws I asked you to move TreatWarningsAsErrors to this file (to be solution-wide) but I didn't ask you to move WarningsNotAsErrors; WarningsNotAsErrors should remain in the .csproj/.fsproj file

Reply:
done